src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1008.PhysicalLocationPropertiesMustBeConsistent_Invalid.sarif

```diff
@@ -11,14 +11,17 @@
               "id": "SARIF1008",
               "name": "PhysicalLocationPropertiesMustBeConsistent",
               "shortDescription": {
-                "text": "If the \"contextRegion\" property is present, the \"region\" property must also be present."
+                "text": "A SARIF 'physicalLocation' object has two related properties 'region' and 'contextRegion'."
```

Comment:
>A SARIF 'physicalLocation' object has two related properties 'region' and 'contextRegion'. [](start = 25, length = 90)

Crud. The first sentence doesn't stand alone. I need to re-author.

Reply:
should i change it back to placeholder? 

---
In reply to: [443866917](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443866917) [](ancestors = 443866917)

Reply:
No, keep this, I will keep it up to date.

---
In reply to: [444583778](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r444583778) [](ancestors = 444583778,443866917)